[docs] Cloud Reorganize Deploy (#10706)
yugabyte/yugabyte-db: docs/content/latest/yugabyte-cloud/cloud-connect/connect-client-shell.md

| sslmode | MITM protection | Notes |
|---|---|---|
| allow | No | Effectively works as _require_ (always uses the SSL connection without verification). |
| prefer | No | Effectively works as _require_ (always uses the SSL connection without verification). |
| require | No | Uses the SSL connection without verification. You do not need to provide the _sslrootcert_ parameter. |
| verify-ca | Yes | Uses the SSL connection and verifies that the server certificate is issued by a trusted certificate authority (CA). Requires the _sslrootcert_ parameter with the path to the cluster certificate. |
| verify-full | Yes | Uses the SSL connection and verifies that the server certificate is issued by a trusted CA and that the requested server host name matches that in the certificate. Requires the _sslrootcert_ parameter with the path to the cluster certificate. |

> If you do not provide an `sslmode`, the connection defaults to `prefer`.

Reviewer: ```suggestion
If you don't provide an `sslmode`, the connection defaults to `prefer`.
```

For detailed steps for configuring popular third party tools, see [Third party tools](../../../tools/). In that section, configuration steps are included for the following tools:

- DBeaver
- DbSchema
- pgAdmin
- SQL Workbench/J
- TablePlus
- Visual Studio Workbench

## Related information

- [ysqlsh](../../../admin/ysqlsh) — Overview of the command line interface (CLI), syntax, and commands.
- [YSQL API](../../../api/ysql) — Reference for supported YSQL statements, data types, functions, and operators.
- [ycqlsh](../../../admin/ycqlsh) — Overview of the command line interface (CLI), syntax, and commands.
- [YCQL API](../../../api/ycql) — Reference for supported YCQL statements, data types, functions, and operators.

## Next steps

- [Add database users](../add-users/)

> - [Connect an application](../connect-application)

Reviewer: Broken link.
